katportalclient/client.py

```diff
@@ -1718,3 +1624,7 @@ class SubarrayNumberUnknown(Exception):
 
 class SensorLookupError(Exception):
     """Raise if requested sensor lookup failed."""
+
+
+def sort_by_timestamp(sample):
```

Comment:
Suggest this is is given a private name, since it isn't useful outside of this file.